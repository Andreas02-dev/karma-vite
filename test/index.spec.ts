import * as output from '@/index';

describe('repo output', () => {
  it('export correctly', () => {
    expect(output).toMatchSnapshot();
  });
});
