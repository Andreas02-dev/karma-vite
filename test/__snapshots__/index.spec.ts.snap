// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`repo output export correctly 1`] = `
Object {
  "default": Object {
    "framework:vite": Array [
      "factory",
      [Function],
    ],
    "middleware:vite": Array [
      "factory",
      [Function],
    ],
    "middleware:vite-before": Array [
      "factory",
      [Function],
    ],
    "vite": Array [
      "factory",
      [Function],
    ],
  },
}
`;
